textproc/asciidoctorj/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           java 1.0
+
+github.setup        asciidoctor asciidoctorj 2.5.2 v
+github.tarball_from archive
+revision            0
+
+categories          textproc java
+platforms           darwin
+license             Apache-2
+supported_archs     noarch
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         \
+    AsciidoctorJ is the official library for running Asciidoctor on the JVM.
+long_description    {*}${description} Using AsciidoctorJ, you can convert \
+    AsciiDoc content or analyze the structure of a parsed AsciiDoc document \
+    from Java and other JVM languages.
+homepage            https://asciidoctor.org
+
+master_sites        https://repo1.maven.org/maven2/org/asciidoctor/asciidoctorj/${version}/
+distname            ${name}-${version}-bin
+worksrcdir          ${name}-${version}
```

Comment:
why not use the defaults from the github PG or is there a reason to override those?

Reply:
I forget to remove this PG. I wanted to build the port from the source but studied the project in more detail understood that build from the source would be difficult. Therefore I  decided to choose the pre-build distribution (which was published by the author on Maven Central).